src/main/java/servlet/OwnerServlet.java

```diff
@@ -0,0 +1,69 @@
+package servlet;
+
+import java.io.IOException;
+import java.sql.Date;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import model.Driver;
+import utils.ParkingDAO;
+import utils.ParkingTollUtils;
+
+@WebServlet(name = "owner",
+			urlPatterns = { "/owner" })
+public class OwnerServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+       
+ 
+    public OwnerServlet() {
+        super();
+    }
+
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		
+	}
+
+	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		
+		String buttonValue = request.getParameter("btn");
+		HttpSession session = request.getSession();
+		RequestDispatcher requestDispatcher = null;
+		
+		switch (buttonValue){
+		
+			case "submit":

```

Comment:
jakby wszystko wewnątrz case'a wyrzucić do innej metody to kod byłby bardziej czytelny